native/src/components/PageDetail.tsx

```diff
@@ -1,32 +1,60 @@
 import React, { ReactElement } from 'react'
 import styled from 'styled-components/native'
 
-import { contentDirection } from '../constants/contentDirection'
+import { InternalPathnameParser } from 'shared'
+
+import buildConfig from '../constants/buildConfig'
+import useNavigate from '../hooks/useNavigate'
+
+const DetailContainer = styled.View`
+  flex-direction: row;
+  color: ${props => props.theme.colors.textColor};
+  align-items: center;
```

Comment:
The direction reversal isn't necessary anymore since it's now a View instead of a Text. So I also removed the tests for that.